argocd_instance/argo-cd-cr-component-properties.adoc

```diff
@@ -26,8 +26,13 @@ include::modules/proc_configuring-by-using-the-ocp-web-console.adoc[leveloffset=
 // Configuring the NotificationsConfiguration CR by using the CLI
 include::modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc[leveloffset=+2]
 
+// Enabling Config Management plugins in an Argo CD CR
+include::modules/con_enabling-config_management_plugins_in_an_argo_cd_cr.adoc[leveloffset=+1]
```

Comment:
Should be re-named with dashes, as file names are separated by dashes and directories are separated by underscores in our repo. There might be some exceptions to this, but generally this is the rule. Note that I've also removed the "stop words" (in, an). 

```suggestion
include::modules/con-enabling-config-management-plugins-argo-cd-cr.adoc[leveloffset=+1]
```

Reply:
Fixed.